pkg/tiles/tile.go

```diff
@@ -88,3 +90,24 @@ func (tile Tile) Rotate(rotations uint) Tile {
 	tile.Features = newFeatures
 	return tile
 }
+
+func (tile Tile) GetTileRotations() []Tile {
```

Comment:
I don't understand what this function does. I'd assume it returns a slice containing all possible rotations of the tile, but why is there a triple nested for loop with two different break statements inside? Is it to eliminate the rotations that would result in the exact same tile? I think a comment explaining exactly what this function returns would be helpful